.github/workflows/wheels.yml

```
@@ -50,8 +50,26 @@ jobs:
       - name: run test
         run: uv run --group test pytest
 
+  cargo-test:
+    name: Run cargo tests
+    needs: pre-commit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)